### ISSUE TITLE
Provide more descriptive error message

### ISSUE THREAD

**Issue by macrovve** [grafana/iot-sitewise-datasource#105]
When we are configuring plugin, there is not much useful information to indicate the exact error. We have to either use inspector or check docker log to guess what is wrong.

**Comment by atifali** [comment]
yes agreed, currently the failure error is too generic and not useful at all. this one takes care of it:
https://github.com/grafana/grafana-plugin-sdk-go/pull/389
